### DISCUSSION TITLE
Install Starfinder game system for newer Foundry version

### DISCUSSION THREAD

**Discussion by Simino** [wildj79/foundryvtt-starfinder#1317] [Q&A] [answered]
Hi everyone

I am completely new to FoundryVTT. My goal is to host a Starfinder campaign but I haven't figured out how to install the Starfinder game system. When I try to install it, I get an error that I require an older version of Foundry.

Is there a way how I can still install it? Respectively, will there be an update soon?

Kind regards,
Simon

PS: Since it looks like this is a community based project, thanks a lot for the great work : ).

**Comment by HarlemHail** [comment, marked as answer]
At this point Starfinder is not compatible with Version 12. If you look here you will find the latest compatible version listed according to Foundry [https://foundryvtt.com/packages/sfrpg/](https://foundryvtt.com/packages/sfrpg/)

**Comment by ian612** [comment]
As of version 0.26, Starfinder is now working with Foundry v12

**Reply by Dany-Deaneris** [reply]
how can we have 0,26 of starfinder i seems not to be able to download above .25.3 version

**Reply by ian612** [reply]
If you are using an older version of Foundry, you may not be able to update. You can see the supported system/Foundry version combinations on the table here: https://foundryvtt.com/packages/sfrpg/

To get the newer starfinder version, you'll need to update to Foundry version 12 (v13 is currently not supported but will be shortly by Starfinder version 0.28 once that's released).

**Reply by Dany-Deaneris** [reply]
i downloaded version 13 build 342 is this possible to update starfinder ?

**Reply by Dany-Deaneris** [reply]
ok, just sees your last answer
